src/client/components/SocialLinks.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import SocialLink from './SocialLink';
+
+export const socialProfiles = [
+  { id: 'facebook', icon: 'facebook', color: '#3b5998', name: 'Facebook' },
```

Comment:
So I noticed we have a lot of colors floating around in our codebase in our javascript files...
would be nice to be keep them in a `constants/styles.js` file and import it based on below see jist here:

https://gist.github.com/jm90m/933417aaf8f1259804957be4df090a91

so `import { COLORS } ` then you get the color from there, i figure out the color name by using this website http://www.color-blindness.com/color-name-hue/

Reply:
I don't think we have many colour definitions in our JS files. Those are only useful in context of social profiles.
Things is we already have file with colours (less) and I don't think we can combine those two.